semantic-conventions/src/tests/data/yaml/metrics.yaml

```diff
@@ -0,0 +1,22 @@
+groups:
+  - id: metric.foo
+    prefix: foo
+    type: metric
+    brief: "This document defines foo."
+    note: >
+        Details about foo.
+
```

Comment:
This is non-functional, right? It looks as if it would define a metric but there is no instrument or unit for it.

Reply:
It serves a purpose of acting as a parent for the other, "concrete" metrics, essentially the root of a tree where related metrics can be grouped under. It's somewhere that attributes which are associated with multiple child metrics can be defined under. 

This is similar to how it works in the trace semconv, for example [http.yaml](https://github.com/open-telemetry/opentelemetry-specification/blob/main/semantic_conventions/trace/http.yaml#L2) has a top-level "http" group which serves as a parent for other spans, while being a place that attributes for all of http can be defined.

Reply:
In that case, the `http` entity of `type: span` does describe parts of an actual span (its attributes), however. I find the dual use of entities of `type: metric` a bit surprising.

Do you think adding a dedicated `type` called `group` or `metric_group` would make sense?
Then all entities of type `metrics` would be an actual metric, consisting of a `metric_name`, `instrument` and `unit`, while still being put under a common root.
Or we leave out the root and have only the individual metrics defined, while they are still combined in the same YAML file.

Reply:
> Do you think adding a dedicated type called group or metric_group would make sense?

Yes, this makes sense, and it was a very simple change to make. See https://github.com/open-telemetry/build-tools/pull/79/commits/e524ab454f344692efe22d90d9beb78338a1edd6. I believe the distinction should be clear now.